### PR TITLE
Update checklist.md

### PR DESCRIPTION
**Pull request by marcellarichmond**
I added this line because this is the second time a partner has stopped the entire pull request process due to thinking they need to test every single command. Hopefully, future partners will read that line and know that is not the case. @ShahafBenYakir let me know if this is fine or if there is a better way to word it. Thank you!!! :)

## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

**Comment by marcellarichmond** [comment]
@mchasepan

File: docs/contributing/checklist.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ If your pack contains an Integration, the integration directory should contain t
 - Command examples file (i.e. `Packs/YourPackName/Integrations/YourIntegrationName/command_examples`): are needed to [autogenerate](../documentation/readme_file#command-examples) the documentation
 - Unit tests file (i.e. `Packs/YourPackName/Integrations/YourIntegrationName/YourIntegrationName_test.py`): must be included to automatically test the code during the review phase. More details [here](../integrations/unit-testing)
 - Unit tests data (i.e. `Packs/YourPackName/Integrations/YourIntegrationName/test_data/*.json`): contain example responses from your product API and are used in [Unit tests](../integrations/unit-testing). [Examples](https://github.com/demisto/content/tree/master/Packs/HelloWorld/Integrations/HelloWorld/test_data) from Hello World.
+- Although we encourage extensive testing, we do **not** enforce a test to each and every function in the code. We suggest you focus on the important ones and the helper functions.
```
Comment (ShahafBenYakir) [inline docs/contributing/checklist.md:44]:
```suggestion
- Unit tests file (i.e. `Packs/YourPackName/Integrations/YourIntegrationName/YourIntegrationName_test.py`): must be included to automatically test the code during the review phase. More details [here](../integrations/unit-testing).  **Note**: Although we encourage extensive testing, we do **not** enforce a test to each and every function in the code. We suggest you focus on the important ones and the helper functions.
- Unit tests data (i.e. `Packs/YourPackName/Integrations/YourIntegrationName/test_data/*.json`): contain example responses from your product API and are used in [Unit tests](../integrations/unit-testing). [Examples](https://github.com/demisto/content/tree/master/Packs/HelloWorld/Integrations/HelloWorld/test_data) from Hello World.
```

**Review by ShahafBenYakir — APPROVED** [review]

**Comment by mchasepan** [comment]
@marcellarichmond this is waiting on you to merge; Does it look ready?

**Comment by marcellarichmond** [comment]
Yes, approved! Sorry I accepted the proposed changes from @ShahafBenYakir and thought that would do it. Let me know what else to do plz!

**Pull request merged**